packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -15,7 +15,7 @@ const defaultProps = {
 
 const variants = {
   size: ['xs', 'sm', 'md', 'lg'],
-  variant: ['primary', 'secondary', 'danger', 'inverse', 'transparent'],
+  variant: ['primary', 'secondary', 'danger', 'inverse', 'transparent', 'link'],
```

Comment:
link is not a variant but different component (Button & LinkButton)

Reply:
`LinkButton` has the same variants (styles) as `Button`. But we need a variant where button will look like link (blue underscored text). If I use `LinkButton` with `transparent` variant, it wouldn't look like a link:

![Screenshot from 2019-12-10 17-55-56](https://user-images.githubusercontent.com/4932851/70540596-99e16d80-1b76-11ea-963b-f772976a3259.png)

Reply:
You said [here](https://github.com/grafana/grafana/pull/20498#discussion_r352539316) 
>This component should be removed we have a LinkButton component already, use the "transparent" variant and restyle it and rename it to "link"

Could you explain, maybe I misunderstood what you want?

Reply:
I agree with @torkelo here. Maybe we could have a `link` variant which is only available for LinkButton that renders what you want here? 

Also, we need to introduce this new variant to the new button components (Forms/Button). I think we could actually introduce it only for the new buttons, and use the new styles instead

Reply:
@dprokop To be clear, do you want to add `link` variant to `LinkButton` only?

Reply:
I think this variant makes sense only in LinkButton